resources/networks.json

```diff
@@ -144,11 +144,11 @@
 		"envDescription": "This environment is used for local development. If you want to have a fully functional environment running in your computer, please visit https://github.com/nevermined-io/tools/",
 		"isProduction": false,
 		"nvm": {
-			"web3ProviderUri": "http://localhost:8545",
-			"marketplaceUri": "http://172.17.0.1:3100",
-			"faucetUri": "http://localhost:3001",
+			"web3ProviderUri": "http://contracts.nevermined.localnet",
+			"marketplaceUri": "http://marketplace.nevermined.localnet",
+			"faucetUri": "http://faucet.nevermined.localnet",
```

Comment:
faucet is deprecated and no longer exists in the sdk

Reply:
removing